src/generic.YAML-tmLanguage

```diff
@@ -0,0 +1,39 @@
+repository:
+    generic:
+        patterns:
+            -
+                begin: '(?=\<(?:[A-Z_]|\*|in|out))'
+                end: '(?<=\>)(?!\>)'
+                patterns:
+                    -
+                        match: '<'
+                        name: punctuation.bracket.angle.begin.kotlin
+                    -
+                        match: '\*'
+                        name: storage.modifier.generic.wildcard.kotlin

```

Comment:
I'd scope this as `entity.name.type.generic.wildcard.kotlin` to be consistent with the class storage scope